plugins/src/android/OptimoveInitProvider.java

```diff
@@ -95,4 +106,33 @@ public int update(@NonNull Uri uri, @Nullable ContentValues values, @Nullable St
         String value = resources.getString(resId);
         return TextUtils.isEmpty(value) ? null : value;
     }
+
+    private DeferredDeepLinkHandlerInterface getDDLHandler() {
+        return (Context context, DeferredDeepLinkHelper.DeepLinkResolution resolution, String link,
+                DeferredDeepLinkHelper.DeepLink data) -> {
+            try {
+                JSONObject dataJson = null;
+                if (null != data) {
+                    JSONObject deepLinkContent = new JSONObject();
+                    if (data.content != null) {
```

Comment:
if i read correct, content is not optional in `DeferredDeepLinkHelper.DeepLink`?

Reply:
I deleted the unnecessary checks for null values

Reply:
I deleted the unnecessary checks for null values

Reply:
`DeferredDeepLinkHelper.DeepLink data` is nullable. i meant data key inside of it. tweaked